src/main/java/ch/njol/skript/entity/SimpleEntityData.java

```diff
@@ -296,7 +296,8 @@ public boolean equals(final @Nullable Object obj) {
 		types.add(new SimpleEntityDataInfo("human", HumanEntity.class, true));
 		types.add(new SimpleEntityDataInfo("damageable", Damageable.class, true));
 		types.add(new SimpleEntityDataInfo("monster", Monster.class, true)); //I don't know why Njol never included that. I did now ^^
-		types.add(new SimpleEntityDataInfo("mob", Mob.class, true)); //Same goes for this one
+		if (Skript.classExists("org.bukkit.entity.Mob")) // Apparently not in 1.9.4
+			types.add(new SimpleEntityDataInfo("mob", Mob.class, true)); //Same goes for this one
```

Comment:
Comment lines aren't for commits or PRs
```suggestion
			types.add(new SimpleEntityDataInfo("mob", Mob.class, true));
```